src/components/Templates/Pages/SignInPage/SignInPage.component.tsx

```diff
@@ -0,0 +1,128 @@
+import { Controller, FieldValues, useForm } from 'react-hook-form';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import VisibilityIcon from '@mui/icons-material/Visibility';
+import VisibilityOffIcon from '@mui/icons-material/VisibilityOff';
+import IconButton from '@mui/material/IconButton';
+import InputAdornment from '@mui/material/InputAdornment';
+import { useState } from 'react';
+import axios, { AxiosResponse } from 'axios';
+import { Link, useNavigate } from 'react-router-dom';
+import { setSessionCookie } from '../../../../utils/cookie';
+import { MODEREN_WALK_USER_ID } from '../../../../config/constants';
+
+const SignInPage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [showPwd, setShowPwd] = useState(false);
+  const navigate = useNavigate();
+
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({ mode: 'onBlur' });
+
+  const onsubmit = async (data: FieldValues) => {
+    setIsLoading(true);
+    try {
+      const res: AxiosResponse = await axios.get(
+        `http://localhost:3001/users?email=${data.email}`
+      );
```

Comment:
@siroshan  Let's add the base url to a .env file so that we can quickly switch it when hosted!